app/design/adminhtml/default/default/template/api/userroles.phtml

```diff
@@ -27,14 +27,13 @@
     <?php foreach ($roles as $role): ?>
     <?php
         $checked = false;
-        foreach( $user_roles->getItems() as $user_role )
-        {
-            if( $user_role->getParentId() == $role->getRoleId() ) {
+        foreach ($user_roles->getItems() as $user_role) {
+            if ($user_role->getParentId() == $role->getRoleId()) {
                 $checked = true;
             }
         }
     ?>
-        <input type="checkbox" value="<?php echo $role->getRoleId(); ?>" <?php echo ($checked ? "checked" : ""); ?> name="roles[]" id="_<?php echo $role->getRoleId(); ?>" /> <label for="_<?php echo $role->getRoleId(); ?>" class="normal"><?php echo $role->getRoleName(); ?></label><br />
+        <input type="checkbox" value="<?php echo $role->getRoleId(); ?>" <?php echo($checked ? "checked" : ""); ?> name="roles[]" id="_<?php echo $role->getRoleId(); ?>" /> <label for="_<?php echo $role->getRoleId(); ?>" class="normal"><?php echo $role->getRoleName(); ?></label><br />
```

Comment:
I found this strange thing: the space is removed:
`<?php echo($checked...`

Reply:
@tmotyl also [found](https://github.com/OpenMage/magento-lts/pull/2847#discussion_r1060178708) this.

Seems to be a bug  ... 

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/4859
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/5950

Reply:
For now, I'd ignore this removed spaces to move forward ...